Pokedex_ios/ViewModel/SortPokemonListViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SortPokemonListViewModel.swift
+//  Pokedex_ios
+//
+//  Created by admin on 7/15/22.
+//
+
+import UIKit
+import Foundation
+
+class SortPokemonListViewModel {
+
+    var pokemons: [PokemonRaw]
+    weak var sortButtonActionDelegateDelegate: SortButtonActionDelegateDelegate?
+
+    init(pokemons: [PokemonRaw]) {
+        self.pokemons = pokemons
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    var closePopup: (() -> Void)?
+        @objc func applyFilters() {
+            closePopup?()
+        }
+    // MARK: Sorting Logic
+    func sortWithSmallestNumberFirst() {
+        pokemons = pokemons.sorted(by: { pokemanRow1, pokemanRow2 in
```

Comment:
typo in pokemon

Reply:
Can you explain a litlle bit more this one please? I'm not sure what I have to correct

Reply:
Corrected :)